documentation/src/docs/streamcuts.md

```diff
@@ -92,4 +92,4 @@ It must be noted that passing ```StreamCut.UNBOUNDED``` to startStreamCut and en
 current head of stream and the current tail of the stream, respectively.
 
 
-We have provided a simple yet illustrative example of using StreamCut [here.](https://github.com/pravega/pravega-samples/tree/{{variable.pravega_samples_verion}}/pravega-client-examples/src/main/java/io/pravega/example/streamcuts)
+We have provided a simple yet illustrative example of using StreamCut [here.](https://github.com/pravega/pravega-samples/tree/v0.3.0/pravega-client-examples/src/main/java/io/pravega/example/streamcuts)
```

Comment:
I'm confused as of why some pointers point to master while this one points to 0.3.0. That's part of the problem I was pointing out previously with respect to versions. It should point to the version with which the docs are released.

Reply:
There are no pointers to master.  The  pointer is either to `v0.3.0` or github repo `https://github.com/pravega/pravega-samples`.

I forgot to fix only this last time while trying few things.

The next time when we release samples this will be updated accordingly. 
So, now 0.3.0 of pravega -> 0.3.0 of pravega-samples
if we have 0.3.1 of  pravega and no release from samples,  
0.3.1 of pravega -> 0.3.0 of pravega-samples, 

So, no broken links or version mismatch.

Reply:
> The next time when we release samples this will be updated accordingly.

how is this going to be updated?

Reply:
We need to do it manually.  Sorry, it's not automatic.
Initially, we agreed upon, having a  variable in `mkdocs.yml`, which is updated will refer the values in all the `.md` files.  But, this is not possible as of now, because of this : https://github.com/mkdocs/mkdocs/issues/304

Reply:
Is there an issue for this? if not, please create it and assign to @nathas1.

Reply:
Created one now https://github.com/pravega/pravega/issues/2767 .